react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+const topics = [
+  { description: 'jdfksadflksajdfjdfksadflksajdfjdfksadflksajdf', name: 'Bulbasaur' },
+  { description: 'jdfksadflksajdfjdfksadflksajdfjdfksadflksajdf', name: 'Charmander' },
+  { description: 'jdfksadflksajdfjdfksadflksajdfjdfksadflksajdf', name: 'Squirtle' },
+  { description: 'jdfksadflksajdfjdfksadflksajdfjdfksadflksajdf', name: 'Pikachu' },
+  { description: 'jdfksadflksajdfjdfksadflksajdfjdfksadflksajdf', name: 'Jigglypuff' }
+];
+
+export default class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: 'false' };
```

Comment:
This doesn't give you enough information in state to determine which description will be visible. It would be better to save the index of the active item. For example:
```JAVASCRIPT
this.state = {
  activeIndex: null
};
```